plugins/cassandra/cassandraembed.cpp

```diff
@@ -432,7 +439,7 @@ class CassandraStatementInfo : public CInterface
     {
         stop();
         futures.kill();
-        delete semaphore;
+        semaphore.clear();
```

Comment:
There was a comment on the line:

// Note - may live on for a while until all futures associated with it have signalled.

which seems to have got lost when I rebased to 5.4 - I will put it back